moveit_kinematics/CMakeLists.txt

```diff
@@ -32,12 +32,12 @@ set(THIS_PACKAGE_INCLUDE_DIRS
 catkin_package(
   LIBRARIES
   INCLUDE_DIRS
-    ${EIGEN3_INCLUDE_DIRS}
     ${THIS_PACKAGE_INCLUDE_DIRS}
   CATKIN_DEPENDS
     pluginlib
     moveit_core
     moveit_ros_planning
+    DEPENDS Eigen3
```

Comment:
OH i didn't submit mine review 10 days ago seems they were pending, got it, thanks @130s